py/desispec/io/util.py

```diff
@@ -187,6 +187,18 @@ def write_bintable(filename, data, header=None, comments=None, units=None,
     """Utility function to write a fits binary table complete with
     comments and units in the FITS header too.  DATA can either be
     dictionary, an Astropy Table, a numpy.recarray or a numpy.ndarray.
+
+    Args:
+        filename: full path to filename to write
+        data: dict or table-like data to write
+
+    Options:
+        header: dict-like header key/value pairs to propagate
+        comments (dict): comments[COLNAME] per column
+        units (dict): units[COLNAME] per column
+        extname (str): extension name for EXTNAME header
+        clobber (bool): if True, overwrite pre-existing file
```

Comment:
I thought we agreed a long time ago to follow the Astropy convention and use `overwrite` instead of `clobber`.